include/circt/Dialect/Calyx/CalyxControl.td

```diff
@@ -0,0 +1,66 @@
+//===- CalyxControl.td - Calyx Control -------------------*- tablegen -*---===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This describes the operations used for the Calyx control execution schedule.
+//
+//===----------------------------------------------------------------------===//
+
+def ControlOp : CalyxContainer<"control", [
+    HasParent<"ComponentOp">
+  ]> {
+  let summary = "Calyx Control";
+  let description = [{
+    The "calyx.control" operation represents the
+    execution schedule defined for the given
+    component, i.e. when each group executes.
+
+    ```mlir
+      calyx.control {
+        calyx.seq {
+          calyx.enable @GroupA
+        }
+      }
+    ```
+  }];
+}
```

Comment:
Should this have a `let regions = ...` declaration? And the SingleBlock/NoTerminator traits?